project/main/business/qhawax.py

```diff
@@ -75,7 +75,7 @@ def getQhawaxStatus():
     name = request.args.get("name")
     try:
         return (
-            str(same_helper.getQhawaxStatus(name))
+            make_response(jsonify(same_helper.getQhawaxStatus(name)))
```

Comment:
This is my second question, @GersonMontesinos. If I keep these endpoints as they are, they return a str, which will then become a HTML without any `<body>` tags. The toolbar can only be used for HTML responses with a `<body>` tag or with JSON responses (now that I added support for them).

There are 3 options here:
1. Do nothing, and you will change this later at some point. The toolbar won't work for these endpoints until you fix it. 
2. Change it to a JSON string, with the code that I wrote here.
3. Change it to a "valid" HTML, by including the missing tags. It would be something like this:

```suggestion
            <html><body><pre>{}</pre></body></html>'.format(
                same_helper.getQhawaxStatus(name)
            )
```

For both options 2 and 3, I would have to change it for all your non-JSON responses, which could take some time. And, more important, your API customers (even the frontend) might need to modify their code in case they are expecting a str() instead of a JSON str or a "valid" HTML (with the extra tags).

What would you like to do? :smile: